x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/scan.cy.ts

```diff
@@ -56,8 +56,8 @@ describe(
             policy = indexedPolicy.integrationPolicies[0];
 
             return enableAllPolicyProtections(policy.id).then(() => {
-              // Create and enroll a new Endpoint host
-              return createEndpointHost(policy.policy_ids[0]).then((host) => {
+              // At this point 8.14.2 is GA and this functionality is not available until 8.15.0
+              return createEndpointHost(policy.policy_ids[0], '8.15.0').then((host) => {
```

Comment:
I assume this will cause failure since `8.15.0` is not GA'ed yet? (unless you wait to merge this PR until it is GA 😄  )